tomcat/setenv.sh

```diff
@@ -1 +0,0 @@
-JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Dxnat.home=/data/xnat/home"
```

Comment:
I'm fine deleting this file, but we do need a way to customize the arguments to pass to tomcat.

Reply:
I totally agree on this. Would an env var in docker-compose.yml suffice to do this?

Reply:
Possibly. I'm not sure how to customize the arguments for tomcat given that we are switching to a pre-built tomcat image. I'll go check on that.

Reply:
I think it will be sufficient to add an environment variable `CATALINA_OPTS` in `docker-compose.yaml`. I guess the value should be the same as what we are removing here, with the change that `xnat.home` should use the `$XNAT_HOME` variable we have already set.